leanpkg/leanpkg/main.lean

```diff
@@ -50,7 +50,8 @@ def make : io unit := do
 manifest ← read_manifest,
 exec_cmd {
   cmd := "lean",
-  args := ["--make"] ++ manifest.effective_path,
+  args := (match manifest.timeout with some t := ["-T", t] | none := [] end) ++
```

Comment:
I think it should **not** be propagated. It will depend on how proofs are structured and which kind of automation is used. Also packages have multiple dependencies, how should we merge two different timeout values?